HtmlDiff/Diff.cs

```diff
@@ -107,14 +125,16 @@ public static string Execute(string oldText, string newText)
         }
 
         /// <summary>
-        /// Builds the HTML diff output
+        /// Builds the HTML diff output (Note: If there is a context specified and the content is equal, then null will be returned)
         /// </summary>
         /// <returns>HTML diff markup</returns>
         public string Build()
         {
             // If there is no difference, don't bother checking for differences
             if (_oldText == _newText)
             {
+                if (LinesContext > 0)
+                    return null;
```

Comment:
I'm not sure returning null is the best option. Perhaps an empty string would be better.